compiler/src/dmd/declaration.d

```diff
@@ -1148,6 +1148,8 @@ extern (C++) class VarDeclaration : Declaration
         bool doNotInferReturn;  /// do not infer 'return' for this variable
 
         bool isArgDtorVar;      /// temporary created to handle scope destruction of a function argument
+        bool inClosure;         /// is inserted into a GC allocated closure
+        bool inAlignSection;    /// is inserted into an aligned section on stack
```

Comment:
Fields in front-end that are only set/read by the back-end.

Reply:
Does that matter?

Reply:
This could be information attached to the backend representation instead, rather than modifying the semantics of an AST node passed to us during codegen.

Shouldn't the code generator treat the finalized AST representation given to it by semantic3 as logically const?

Reply:
The backend symbols are not created at the time the stack alignment code is generated. There's no place to hang the information.

Reply:
You can compute it in-flight during the glue/codegen pass when you have the full picture available, but whatever.